### DISCUSSION TITLE
Adding Dots to Homepage

### DISCUSSION THREAD

**Discussion by SingularDave** [thuliteio/doks#966] [General]
Hello,

The dots on the Doks homepage look great. I'm trying to add that style to my site. I read over https://github.com/h-enk/doks/discussions/104, but I still need to understand how to apply that style to my site. Please give me instructions on how to do this. I'm new to scss. Thank you!

**Comment by h-enk** [comment]
This is how I did it for the getdoks.org website:

- the HTML: https://github.com/h-enk/getdoks.org/blob/master/layouts/index.html#L17-L19
- the SCSS: https://github.com/h-enk/getdoks.org/blob/master/assets/scss/common/_global.scss#L234-L252

**Reply by SingularDave** [reply]
Works perfectly. Thanks @h-enk !
